src/prowjobsscraper/equinix.py

```diff
@@ -0,0 +1,66 @@
+import logging
+from typing import Final, Optional
+
+from google.cloud import exceptions, storage  # type: ignore
+
+from prowjobsscraper import utils
+from prowjobsscraper.prowjob import EquinixMetadata, ProwJob, ProwJobs
+
+logger = logging.getLogger(__name__)
+
+
+class EquinixExtractor:
+    """
+    EquinixExtractor parses the Equinix metadata generated by the gather step in the assisted-baremetal workflow.
```

Comment:
I see you're handling ``assisted-baremetal-operator-workflow`` as well, no?

Reply:
:man_facepalming: